test/over_react/util/rem_util_test.dart

```diff
@@ -105,39 +131,53 @@ main() {
         expect(toPx(new CssValue.parse('0.75rem')), new CssValue(15, 'px'));
       });
 
-      test('converts an int value (treated as rem) to px', () {
+      test('converts nums (treated as rem) to px', () {
         expect(toPx(3), new CssValue(60, 'px'));
+        expect(toPx(1.01), new CssValue(20.2, 'px'));
       });
 
-      test('converts a double (treated as rem) to px', () {
-        expect(toPx(1.01), new CssValue(20.2, 'px'));
+      test('does not convert nums when `treatNumAsPx` is true', () {
+        expect(toPx(3, treatNumAsPx: true), new CssValue(3, 'px'));
+        expect(toPx(1.01, treatNumAsPx: true), new CssValue(1.01, 'px'));
+      });
+
+      test('gracefully handles a px String, not doing any conversion', () {
+        expect(toPx('1334px'), new CssValue(1334, 'px'));
+      });
+
+      test('gracefully handles a px CssValue, not doing any conversion', () {
+        expect(toPx(new CssValue(1334, 'px')), new CssValue(1334, 'px'));
+      });
+
+      test('gracefully passes through `null`', () {
+        expect(toPx(null), null);
       });
 
       test('throws when passed an invalid value', () {
         expect(() => toPx(new Object()), allOf(
             throwsArgumentError,
-            throwsA(hasToStringValue(contains('must be a num or a String rem value'))))
+            throwsA(hasToStringValue(contains('must be a rem num or a String px/rem value'))))
         );
       });
 
       test('throws when passed a malformed CSS value string', () {
         expect(() => toPx(''), allOf(
             throwsArgumentError,
-            throwsA(hasToStringValue(contains('must be a num or a String rem value'))))
+            throwsA(hasToStringValue(contains('must be a rem num or a String px/rem value'))))
         );
       });
 
-      test('throws when passed a CSS value string with a unit other than px', () {
+      test('throws when passed a CSS value string with a unit other than px/rem', () {
         expect(() => toPx('1em'), allOf(
             throwsArgumentError,
-            throwsA(hasToStringValue(contains('must be a num or a String rem value'))))
+            throwsA(hasToStringValue(contains('must be a rem num or a String px/rem value'))))
         );
       });
 
-      test('throws when passed a CssValue instance with a unit other than px', () {
+      test('throws when passed a CssValue instance with a unit other than px/rem', () {
         expect(() => toPx(new CssValue.parse('1em')), allOf(
             throwsArgumentError,
-            throwsA(hasToStringValue(contains('must be a num or a String rem value'))))
+            throwsA(hasToStringValue(contains('must be a rem num or a String px/rem value'))))
         );
       });
```

Comment:
Is it possible that Codecov actually _wasn't_ bae?
https://github.com/Workiva/over_react/pull/7/commits/cfdd19985078ed15d046f43466b478228f5d02f7